components/create_comment/create_comment.jsx

```diff
@@ -592,6 +607,29 @@ class CreateComment extends React.PureComponent {
             setTimeout(() => {
                 this.setState({errorClass: null});
             }, Constants.ANIMATION_TIMEOUT);
+        }
+
+        const isReaction = Utils.REACTION_PATTERN.exec(draft.message);
+        if (isReaction && this.props.emojiMap?.has(isReaction[2])) {
+            let isIncomingReactionPresentForLatestPost = this.props.reactionsByEmojiNameForLatestPost?.indexOf(isReaction[2]) > -1;
+            if (
+                !isIncomingReactionPresentForLatestPost &&
+                isReaction[1] === '+' &&
+                this.props.latestPostReactionsCount >= Constants.EMOJI_REACTIONS_LIMIT
+            ) {
+                const errorMessage = (
+                    <FormattedMessage
+                        id='create_post.reaction_limit_message'
+                        defaultMessage='Reaction limit exceeded for this message.'
+                    />
+                );
+
+                setTimeout(() => this.handlePostError(errorMessage), Constants.REACTION_LIMIT_MESSAGE_TIMEOUT);
```

Comment:
I see another one on top existing in the code but can you clear this timer on unmount?